makefiles/Makefile.build-lwip2

```diff
@@ -81,4 +81,4 @@ endif
 
 clean:
 	rm -rf $(BUILD) $(LWIP_LIB) $(AUTO)
-	cd lwip2-src; rm -f src/.patched src/core/*.c.orig src/core/ipv6/*.c.orig; git checkout -- src;
+	cd lwip2-src; git checkout -f -- src; git clean -f -- src;
```

Comment:
- `git clean` removes untracked `*.orig` and `.patched`
- just `git checkout` did not remove patched files for me when running different prefixes for tests, `-f` did the trick